src/plugins/dashboard/public/dashboard_actions/filters_notification_popover_contents.tsx

```diff
@@ -60,16 +60,14 @@ export function FiltersNotificationPopoverContents({ context }: FiltersNotificat
   });
 
   return (
-    <>
-      {isLoading ? (
-        <EuiLoadingContent lines={3} />
-      ) : (
-        <EuiForm
-          component="div"
-          css={css`
-            min-width: 300px;
-          `}
-        >
+    <EuiForm
+      component="div"
+      css={css`
+        min-width: 300px;
+      `}
+    >
+      <EuiSkeletonText isLoading={isLoading} lines={3}>
+        <>
```

Comment:
Is `<>` needed? Doesn't EuiSkeletonText already provide a wrapping element?

Reply:
Oh, good catch! I just assumed `EuiSkeletonText` expected only one child component but seems like that's not the case 👍 Fixed in 8c23a1299411f5f7d7800e4cf37587fa7312efb0